### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by wind91725** [gajanlee/V-net#2]
self.content_prob = tf.nn.sigmoid(vars["w_1_c"] * tf.nn.relu(vars["w_2_c"]*self.point_logits))

vars["w_1_c"] ，vars["w_2_c"]没定义呀，而且后面是乘self.point_logits？ 根据论文 应该乘 self.passage_encoding吧？
